java/sage/MediaFile.java

```diff
@@ -3516,7 +3438,7 @@ private void checkForTVFileConversion(Show s)
   private void checkForTVFileConversion(String externalID)
   {
     // Check for conversion to a TV file. This is true if it uses TMS data for the Show IDs
-    if ((!Sage.EMBEDDED || SageConstants.PVR)
+    if (SageConstants.PVR
```

Comment:
(!Sage.EMBEDDED || SageConstants.PVR) always evaluates true in the old code. We can remove `SageConstants.PVR`.